core/src/main/scala/org/apache/spark/executor/ProcfsMetricsGetter.scala

```diff
@@ -171,7 +172,8 @@ private[spark] class ProcfsMetricsGetter(procfsDir: String = "/proc/") extends L
       val pidDir = new File(procfsDir, pid.toString)
       def openReader(): BufferedReader = {
         val f = new File(new File(procfsDir, pid.toString), procfsStatFile)
-        new BufferedReader(new InputStreamReader(new FileInputStream(f), Charset.forName("UTF-8")))
+        new BufferedReader(new InputStreamReader(new FileInputStream(f),
+          StandardCharsets.UTF_8.name()))
```

Comment:
Ditto. Let's import `java.nio.charset.StandardCharsets.UTF_8` instead of splitting this into two lines?

Reply:
Actually you can pass the `Charset` directly here, and avoid `name()`. That would be a little better. Same for any `InputStreamReader`